test/mpi/test_mpi.cc

```diff
@@ -72,13 +90,22 @@ UccTestMpi::UccTestMpi(int argc, char *argv[], ucc_thread_mode_t _tm, int is_loc
     UCC_CHECK(ucc_context_config_read(lib, NULL, &ctx_config));
     UCC_CHECK(ucc_context_create(lib, &ctx_params, ctx_config, &ctx));
     ucc_context_config_release(ctx_config);
+    setenv("UCC_TL_UCP_TUNE", "alltoall:0-inf:@onesided", 1);
+    UCC_CHECK(ucc_lib_config_read(NULL, NULL, &lib_config));
+    UCC_CHECK(ucc_init(&lib_params, lib_config, &onesided_lib));
+    ucc_lib_config_release(lib_config);
+    UCC_CHECK(ucc_context_config_read(onesided_lib, NULL, &ctx_config));
+    UCC_CHECK(ucc_context_create(onesided_lib, &onesided_ctx_params, ctx_config,
```

Comment:
So, the 1-sided ctx is always created? Even if the one-sided switch was not requested?

Reply:
fixed. should only be created when testing one-sided paths now